docs/release_process.rst

```diff
@@ -15,79 +14,112 @@ number occurs.
 Create and Merge Version Update Branch
 --------------------------------------
 
-Patch Release
-^^^^^^^^^^^^^
-A patch release should only contain fixes for bugs or issues with an existing
-release.
-No new features or functionality should be introduced in a patch release.
-As such, patch releases should only ever be based on an existing release point.
+Major and Minor Releases
+^^^^^^^^^^^^^^^^^^^^^^^^
+Major and minor releases may add one or more trivial or non-trivial features
+and functionalities.
 
-1. Create a new branch off of the appropriate ``vX.Y.Z`` tag named something
-   like ``release-patch-{NEW_VERSION}``, where ``NEW_VERSION`` is an increase
-   in the ``Z`` version component.
+1. Create a new branch off of the ``master`` named something like
+   ``update-to-v{NEW_VERSION}``, where ``NEW_VERSION`` is the new ``X.Y``
+   version.
 
-   a. Use ``poetry version patch`` to increase the patch value appropriately in
-      the :file:`pyproject.toml` file.
+   a. Use the ``scripts/update_release_notes.sh`` script to update the project
+      version number, create ``docs/release_notes/v{NEW_VERSION}.rst``, and add a
+      new pending release notes stub file.
 
-   b. Rename the ``docs/release_notes/pending_patch.rst`` file to
-      ``docs/release_notes/v{VERSION}.rst``, matching the new version value.
-      Add a descriptive paragraph under the title section summarizing this
-      release.
+      .. code-block:: bash
 
-   c. Add a reference to the new release notes RST file in
-      ``docs/release_notes.rst``.
+         $ ./scripts/update_release_notes.sh minor
 
-   d. In a separate commit, add back a blank pending release notes file stub.
-      See `Stub Pending Notes File`_.
+   b. Add a descriptive paragraph under the title section of
+      ``docs/release_notes/v{NEW_VERSION}.rst`` summarizing this release.
 
-2. Create a pull/merge request for this branch with master as the merge target.
-   This is to ensure that everything passes CI testing before making the
-   release. If there is an issue, then branches should be made and merged into
-   this branch until the issue is resolved.
+2. Push the created branch to the upstream repository, not your fork (this is
+   an exception to the normal forky workflow).
 
-3. Tag branch (see `Tag new version`_ below ) after resolving issues and before
-   merging into ``master``.
+3. Create a pull/merge request for this branch with ``release`` as the merge
+   target. This is to ensure that everything passes CI testing before making
+   the release. If there is an issue, then topic branches should be made and
+   merged into this branch until the issue is resolved.
 
-4. Merge version bump branch into ``master`` branch.
+4. Get a positive review.
 
-5. `Create new version release to PYPI`_.
+5. Merge version bump branch into the ``release`` branch.
```

Comment:
Is this the same as for patches? i.e.

```
5. Merge the pull/merge request into the ``release`` branch.
```

I find the patch instructions a bit simpler to parse (but could just be me).

Reply:
Oops, that item is also supposed to be symmetric between major/minor/patch. The line present in the patch version is what I meant to propagate up.